- fix locale for emails that are triggered by a paypal ipn message

nicobea opened the pull request:
Hi,

I noticed that emails that are triggered by an asynchronous PayPal IPN message were not in the correct locale. To fix that I just made some updates to your Emails class inspired from WooCommerce /includes/gateways/paypal/includes/class-wc-gateway-paypal-ipn-handler.php.

Best regards and thanks for the great work!

hyyan commented:
@nicobea thank you for the PR , but your method name does not follow the code standard of this plugin could you please change to something like `getOrderIDFromIPNRequest` , thanks you :+1:

hyyan commented:
@nicobea  I am using  http://cs.sensiolabs.org/ for plugin code standard , you can use it.

nicobea commented:
@hyyan, I just made a new commit and I changed the method name as you suggest. I've also removed some extra spaces that are not compliant with the coding standards you use.
Thank you.

hyyan commented:
@nicobea Thank you for PR :smile:
